docs/user/resources/04-30-10-aws-vpc-peering.md

```diff
@@ -62,6 +62,11 @@ Use the following table to identify Cloud Manager principal based on your Kyma l
 
 3. Attach the **CloudManagerPeeringAccess** policy to the **CloudManagerPeeringRole**:
 
+## Required Actions in the Remote Project
+
+Before creating the VPC peering, please tag your AWS account VPC with the Kyma shoot name tag.  
+For more information, check the [Create Virtual Private Cloud Peering in Amazon Web Services](../tutorials/01-30-10-aws-vpc-peering.md) tutorial.
```

Comment:
Where is the tagging on the tutorial? Is it step 7? Can we maybe describe how to do only the tagging here? When we redirect a user to the tutorial there are many more steps than just the tagging and it might be confusing.